### PR TITLE
feat(MESSAGE): support persistent Snackbars and allow to know if the button was pressed

### PR DESCRIPTION
**Pull request by atabel**
fixes https://github.com/Telefonica/webview-bridge/issues/117

jira: https://jira.tid.es/browse/WEB-1599

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -411,35 +411,41 @@ nativeAlert: ({
...
--   `buttonText` property is ignored in iOS.
-
 <img height="550" src="doc/webview-bridge-message-ios.png"><img height="550" src="doc/webview-bridge-message-android.png">
 
 ```ts
 nativeMessage: ({
         message: string;
-        duration?: number; // milliseconds
-        buttonText?: string; // Android only
+        duration?: 'PERSISTENT' | 'FIVE_SECONDS' | 'TEN_SECONDS';
```
Comment (pladaria) [inline README.md:427]: isn't this a breaking change?

Comment (pladaria) [reply README.md:427]: Ok, I see that the apps were ignoring this param but this isn't implementing the proposed solution in #117

image

Comment (pladaria) [reply README.md:427]: Ok, I see that the implementation allows the `number` type. Which one is correct?

File: src/dialogs.ts [hunk 2]
```diff
@@ -50,21 +54,44 @@ export const nativeMessage = ({
...
+    withDismiss,
 }: {
     message: string;
-    duration?: number; // milliseconds
+    /**
+     * By default, the snackbar auto dismisses after 5s or 10s (if it has buttonText)
+     * PERSISTENT: the snackbar won't dismiss without user interaction
+     * FIVE_SECONDS: the snackbar will dismiss after 5 seconds
+     * TEN_SECONDS: the snackbar will dismiss after 10 seconds
+     * number: deprecated, this is ignored by native app
```
Comment (yamal-alm) [inline src/dialogs.ts:65]: I think we should add which default values are used if a numeric duration is passed to the method:
If `buttonText` is `null` then `FIVE_SECONDS`, else `TEN_SECONDS`

And maybe is it worthy to mention that:
- `FIVE_SECONDS` will be set instead when the method receives no `buttonText` and `TEN_SECONDS` as its duration
- `TEN_SECONDS` will be set instead when the method recieves a non-null `buttonText` and `FIVE_SECONDS` as its duration

**Review by yamal-alm — APPROVED** [review]
nice 👍 just a suggestion about a comment

**Review by idenjoe — APPROVED** [review]

File: src/dialogs.ts (hunk 2 above)
Comment (atabel) [reply src/dialogs.ts:65]: hmmm... I have a doubt with this, then why do we need those enum values? I mean, when can I use `TEN_SECONDS` or `FIVE_SECONDS`?, it seems like those values will be always ignored, wont't they?

Comment (atabel) [reply src/dialogs.ts]: removed `TEN_SECONDS` and `FIVE_SECONDS` from the enum

**Pull request merged**